src/server/managers/AppListenerManager.ts

```diff
@@ -578,4 +581,24 @@ export class AppListenerManager {
             }
         }
     }
+
+    private async executeBlockitAction(data: IBlockitAction): Promise<void> {
+        console.log('executeBlockitAction ->', data);
+        for (const appId of this.listeners.get(AppInterface.IBlockitActionHandler)) {
+            const app = this.manager.getOneById(appId);
+
+            console.log('appId ->', appId);
+
+            if (app.hasMethod(AppMethod.BLOCKIT_ACTION)) {
+                console.log('hasMethod');
+                await app.call(AppMethod.BLOCKIT_ACTION,
+                    data,
```

Comment:
use `const cfBlockitAction = Utilities.deepCloneAndFreeze(data)` to deep clone and freeze the copy of data first to avoid apps mutate the data directly